### DISCUSSION TITLE
ACAP 4 axPTZ support.

### DISCUSSION THREAD

**Discussion by oferardi** [AxisCommunications/discussions#112] [ACAP] [answered]
Hi

My current product is based on axPTZ api capabilities.

In the ACAP 4 page is written that the support : 

"Not yet decided, i.e. may be replaced or moved as is to Native SDK."

Is it going to be supported and when? I need this support in order to use axis products for current and future customers otherwise i need to develop my own camera..

Please advise asap

Thanks

**Comment by mattias-kindborg-at-work** [comment, marked as answer]
Hi @oferardi. As you've stated, there is no support for this and it has not been decided whether any support for this will be implemented in the future. I'll make sure to forward your requirement on axPTZ to the development team.

Is your ACAP app currently implemented in C or C++, and given the option to, would you see yourself preferring to implement your app in any other language given you had the chance and API to do so?

**Reply by oferardi** [reply]
Hi Mattias

  You wrote that it's not yet decided if any support will be implemented.

Please notice that this is what is written the developer website:

axPTZ API Not yet decided, i.e. may be replaced or moved as is to Native SDK.

C++ is preferred.

This functionality is critical for as my current product is based on these capabilites..

Thanks

Ofer

**Reply by mattias-kindborg-at-work** [reply]
@oferardi I haven't seen a timeline for this feature. That basically tells me that perhaps this will be implemented, perhaps it wont. It comes down to priority.

Thanks for the feedback regarding your choice of language, and I'll make sure to forward your thoughts regarding the importance of this API.
